bundles/pubsub/pubsub_serializer_avrobin/src/pubsub_avrobin_serializer_impl.c

```diff
@@ -49,10 +49,10 @@ struct pubsub_avrobin_serializer {
     log_helper_t *loghelper;
 };
 
-static celix_status_t pubsubMsgAvrobinSerializer_serialize(void *handle, const void *msg, void **out, size_t *outLen);
-static celix_status_t pubsubMsgAvrobinSerializer_deserialize(void *handle, const void *input, size_t inputLen, void **out);
-static void pubsubMsgAvrobinSerializer_freeMsg(void *handle, void *msg);
-
+static celix_status_t pubsubMsgAvrobinSerializer_serialize(void *handle, const void *msg, struct iovec** output, size_t* outputIovLen);
```

Comment:
Shouldn't all those changes be in a different PR?

Reply:
I agree, but for testing / integrating on real system  with very high troughput I needed everything in this PR.